operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -300,3 +300,57 @@ data:
         expr: sum(rate(node_network_receive_drop_total{job=\"node-exporter\", device!=\"lo\"}[1m])) without(device)
       - record: instance:node_network_transmit_drop_excluding_lo:rate1m
         expr: sum(rate(node_network_transmit_drop_total{job=\"node-exporter\", device!=\"lo\"}[1m])) without(device)
+  metrics_rules.yaml: |
+    - name: NamespaceHighCPUUsage
+      expr: (sum(kube_resourcequota{resource=\"limits.cpu\", type=\"used\"}) by (namespace, resourcequota) / sum(kube_resourcequota{resource=\"limits.cpu\", type=\"hard\"}) by (namespace, resourcequota)) * 100 > 70
+      for: 2m
+      metrics_list:
+        container_cpu_cfs_periods_total
+        container_cpu_cfs_throttled_periods_total
+        kube_pod_container_resource_limits
+        kube_pod_container_resource_requests
+        kube_resourcequota
+        mixin_pod_workload
+        namespace_workload_pod:kube_pod_owner:relabel
+        node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate
+        node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate
+
+    - name: NamespaceHighMemoryUsage
+      expr: (sum(kube_resourcequota{resource=\"limits.memory\", type=\"used\"}) by (namespace, resourcequota) / sum(kube_resourcequota{resource=\"limits.memory\", type=\"hard\"}) by (namespace, resourcequota)) * 100 > 70
+      for: 2m
+      metrics_list:
+        - container_memory_cache
+        - container_memory_rss
+        - container_memory_swap
+        - container_memory_working_set_bytes
+        - kube_pod_container_resource_limits
+        - kube_pod_container_resource_requests
+
+    - name: SNOHighCPUUsage
+      expr: cluster:container_cpu_usage:ratio * 100 > 70
+      for: 2m
+      selector:
+        clusterType: "SNO"
+      metrics_list:
+        container_cpu_cfs_periods_total
+        container_cpu_cfs_throttled_periods_total
+        kube_pod_container_resource_limits
+        kube_pod_container_resource_requests
+        kube_resourcequota
+        mixin_pod_workload
+        namespace_workload_pod:kube_pod_owner:relabel
+        node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate
+        node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate
+
+    - name: SNOHighMemoryUsage
+      expr: cluster:memory_usage:ratio * 100 > 70
+      for: 2m
+      selector:
+        clusterType: "SNO"
+      metrics_list:
+        - container_memory_cache
+        - container_memory_rss
+        - container_memory_swap
+        - container_memory_working_set_bytes
+        - kube_pod_container_resource_limits
+        - kube_pod_container_resource_requests
```

Comment:
where is the metrics_list come from?
ideally, they should cover all the pod/container level metrics in dashboards below:
Kubernetes / Compute Resources / Namespace (Pods)
Kubernetes / Compute Resources / Namespace (Workloads)
Kubernetes / Compute Resources / Nodes (Pods)
Kubernetes / Compute Resources / Pod
Kubernetes / Compute Resources / Workload

Reply:
The metrics that are listed are from those dashboards. Here is the comment that captures the metrics: https://github.com/stolostron/backlog/issues/19141#issuecomment-1050186409.